smart-contracts/test/test_bridgeBankLock.js

```diff
@@ -147,6 +175,19 @@ describe("Test Bridge Bank", function () {
     });
 
     it("should allow user to multi-lock ERC20 tokens with multiLockBurn method", async function () {
+      // Add the tokens into white list
+      await state.bridgeBank.connect(operator)
+        .updateEthWhiteList(state.token1.address, true)
+        .should.be.fulfilled;
+
+      await state.bridgeBank.connect(operator)
+        .updateEthWhiteList(state.token2.address, true)
+        .should.be.fulfilled;
+
+      await state.bridgeBank.connect(operator)
```

Comment:
Seeing how you have to send 3 tx's here in order to whitelist 3 separate tokens seems wasteful. This should be a separate PR but we need to have a bulkWhitelist function that handles eth assets to whitelist. This will save us gas on bulk updates.